.github/workflows/build.yml

```diff
@@ -41,6 +41,6 @@ jobs:
         make -j4
         sudo make install
     - name: Build
-      run: python3 setup.py install --user --sundials-home=/usr --blas-home=/usr/lib/x86_64-linux-gnu/ --lapack-home=/usr/lib/x86_64-linux-gnu/ --superlu-home=/usr --extra-fortran-compile-flags="-fallow-argument-mismatch"
+      run: python3 setup.py install --user --sundials-home=/usr --blas-home=/usr/lib/x86_64-linux-gnu/ --lapack-home=/usr/lib/x86_64-linux-gnu/ --superlu-home=/usr --extra-fortran-compile-flags="-fallow-argument-mismatch --std=legacy"
```

Comment:
then `--extra-fortran-compile-flag` can be dropped completely isnt it ?
wasnt it the point of this PR ?

Reply:
I can be dropped here, right.

Added it as the recommended default value to the repeated issues with this recently. Was still planning to leave the principle mechanism to pick different/extra flags in place.